examples/support/mod.rs

```diff
@@ -62,6 +62,7 @@ pub trait ApplicationContext {
     fn new(display: &Display<WindowSurface>) -> Self;
     fn update(&mut self) { }
     fn handle_window_event(&mut self, _event: &WindowEvent, _window: &winit::window::Window) { }
+    fn window_title() -> &'static str { "Glium example" }
```

Comment:
Can this be an associated constant instead? No need for a function. That also lets you avoid typing the `'static` every time.

Reply:
Sure, that makes sense. I've amended the last commit. Couldn't get rid of the lifetime specifier though.

Reply:
Thanks!

> Couldn't get rid of the lifetime specifier though.

Ahh yeah makes sense, there can be lifetimes in traits and such so it can't be omitted from there.